test/automl/test_lexiflow.py

```diff
@@ -0,0 +1,40 @@
+from flaml import AutoML
+from flaml.data import load_openml_dataset
+
+
+def _test_lexiflow():
+
+    X_train, X_test, y_train, y_test = load_openml_dataset(
+        dataset_id=179, data_dir="test/data"
+    )
+
+    lexico_objectives = {}
+    lexico_objectives["metrics"] = ["val_loss", "pred_time"]
+    lexico_objectives["tolerances"] = {"val_loss": 0.01, "pred_time": 0.0}
+    lexico_objectives["targets"] = {"val_loss": 0.0, "pred_time": 0.0}
+    lexico_objectives["modes"] = ["min", "min"]
+
+    automl = AutoML()
+    settings = {
+        "time_budget": 100,
```

Comment:
shorter for code correctness check; 
for quality check it's ok to be long, but for that purpose, there needs to be assertion about the performance, and such test is only needed once among all the test environments. See test_notebook_example.py for an example.